src/screens/Home/Home.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { Box, Image, Text } from 'grommet'
+import Kelp from 'images/kelp.png'
+import Title from 'images/title.svg'
+import ResponsiveImage from 'components/ResponsiveImage'
+import styled from 'styled-components'
+import ChooseLocation from './components/ChooseLocation'
+import { ZooniverseLogotype } from '@zooniverse/react-components'
+
+const StyledText = styled(Text)`
+  font-family: Neuton;
+`
+
+export default function Home() {
+  return (
+    <Box background='sand' direction='row'>
+      <Box
+        basis='1/4'
+        border={{ color: 'kelp', side: 'right', size: 'large' }}
+      >
+        <Image a11yTitle='Kelp Background' fit='cover' src={Kelp} />
+      </Box>
+
+      <Box gap='medium'>
+        <Box
+          margin={{ horizontal: 'medium', top: 'large' }}
+          width={{ max: '40rem' }}
+        >
+          <ZooniverseLogotype />
+          <ResponsiveImage a11yTitle='Floating Forests: Mapping Visualizations' src={Title} />
```

Comment:
Instead of loading in the svg as an image, make it a component and use the same approach we use with the `ZooniverseLogotype` to make it accessible with the `title` tag and `id` and `aria-labelledby` attributes.

Reply:
Just curious. Are we losing anything by using an image with `a11yTitle` here? This will take a good bit of work to implement to have the SVG remain responsive with page resize, which was a difficulty I originally had using the plain Grommet `Image` component.

Reply:
Well... actually. I might've just got it working.

Reply:
@eatyourgreens might have some more experience with this, but I think the read out will be confusing by having the screen reader say there's an image and then the alt text be the words. It's not really clear what it means or what it is? Ideally this shouldn't be an image at all, but since it's stylized text, the middle ground is the approach we took with the `ZooniverseLogotype` and SVG using the title HTML tag, so it's read out like text rather than an image.

Reply:
Is this the page heading? If so, there should be a `h1` somewhere.

Something like this `<h1><img src="/images/fancyText.jpg" alt="Page Heading"></h1>` would get read out more-or-less as a page heading by screenreaders, but @srallen is right, it looks like the image is being used where styled text would be more appropriate. If I'm understanding this correctly, can the text here be resized by the page zoom controls in the browser? If not, I'm pretty sure that would be a WCAG AA failure.
https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-text-images.html

Reply:
I'm not totally clear why this isn't text, but might have to do with the availability of the font?

Reply:
Oh, that takes me back to Dreamweaver and slicing up images of text because the web only had two fonts: Times Roman and Arial.